### ISSUE TITLE
Standby EP doesn't appear to be functional

### ISSUE THREAD

**Issue by smw6180** [Alumian/CEPGP#24]
Looking through the options there is a section for standby ep.  However I see no way to utilize it.  Perhaps I'm misunderstanding what is meant here by standby, but I was hoping to be able to create a standby list of players (not necessarily everyone in guild) via a !queue option or something similar.  I've looked through the code and don't see anything in it pertaining to a standby list, and given our guild's content we can't assume everyone is on standby.  Does this add-on provide this functionality?

Thanks!

**Comment by Alumian** [comment]
Hi,

The add-on does currently support standby EP. Essentially what the existing function does is it will award a percentage of all EP awarded to players on the delegated list. I.e. if you defeat a boss worth 10 EP for example and you have a player on the standby list and the standby percentage is 10%, then the standby player will receive 1 EP.
The standby percentage can be specified in the options menu. Is that the kind of feature you're looking for?

**Comment by smw6180** [comment]
I don't see a way to add a player to a standby list at all.  Am I missing it?

**Comment by Alumian** [comment]
No, you're not missing it. Turns out it's a feature I never added. Please see my edit in my last post. Just to confirm, did you just want the functionality to add certain players to the list instead of having rank delegation?

Thanks

**Comment by smw6180** [comment]
Ideally a player would whisper !standby to one of the raid leaders and be put on a standby list...players on that list would get standby ep based on the options selected that you already have.  Would be nice to have a way to remove players from the standby list as well if possible.

**Comment by smw6180** [comment]
Oh and thank you for checking into it.  I really do appreciate it.  :)

**Comment by Alumian** [comment]
You're welcome :)
This doesn't sound like a difficult feature to add but I feel like it would be more efficient to have the raid leader add people to a list rather than having people opt in. This would prevent unwanted people from adding themselves to the standby ep list without the intent of the guild officers. It would also be way easier to write lol

Let me know what your thoughts are on this matter

Thanks.

**Comment by Alumian** [comment]
I'll put in an option for both :)
Oh wow thank you for your support ^_^ I've never had anyone that wanted to donate before haha
Definitely not required but if you'd like to donate to me the link is paypal.me/Alumian

I'll close the ticket once the feature has been implemented.
Feel free to post back any time if you have any more specific requirements :)

**Comment by smw6180** [comment]
As far as I know that's the main thing they want:  !standby to auto-add to a list, with the option for the user to remove themselves (maybe !standby remove or something) and with the officers having the ability to remove users from standby as well.  Donation has been sent.  :) I didn't expect you'd be 'expecting' it but I feel that since I'm asking you to add special features into your already excellent software it's only fair that I compensate you for the time it takes to work on it.  If I had to do it it'd probably take 6 weeks or more. :)

**Comment by Alumian** [comment]
Hi smw6180,

I just wanted to let you know that for the most part, I have implemented this feature but I just have to work out some finer details and additional functionality.
I wanted to apologise as well because my motherboard is defective and will need to return it to the manufacturer for a replacement, so unfortunately my computer isnt going to be operational for 1-2 weeks.

I expect the feature should not take too much longer to implement once my computer is running again. I'll keep you posted.

Thanks

**Comment by smw6180** [comment]
Thanks for the update.  Appreciate you getting back to me on it. :)

**Comment by Alumian** [comment]
Hey man,

Sorry I haven't chimed in in a while. The motherboard came back, turns out that's not the problem >.>
Thought it could be the RAM so that was sent off about a week or 2 ago for repair so now I'm just waiting on that to come back.

Sorry for the delay. I'll let you know when my pc is back up and running.

**Comment by Alumian** [comment]
Alright so fun times. I just got an email advising that the RAM has been sent to a partner company for repair and I can expect it back in the next 21-28 days.

**Comment by smw6180** [comment]
Ouch.  You sound like you're getting clobbered with bad computer luck.  Thanks for the update.

**Comment by Alumian** [comment]
Okay so I finally have the RAM back, had to reinstall the OS, computer turns on, the screen works, the keyboard does stuff, I should be able to get back to the add-on soon. Sorry for the delays!

**Comment by Alumian** [comment]
Hi smw6180,
Sorry for the delay. I've pushed my work to the [beta branch](https://github.com/Alumian/CEPGP/archive/beta.zip). Please let me know what you think of these changes. If the addon hasn't caught fire in 1 month from now then I will merge this to the master branch and mark this ticket as resolved.

Please be aware that after I merge the beta branch, this project will be marked as unsupported and future updates will not be provided unless I return to classic warcraft.

Thanks,
Alumian
